internal/descriptor/services_test.go

```diff
@@ -1324,9 +1331,11 @@ func TestExtractServicesWithDeleteBody(t *testing.T) {
 		for _, src := range spec.srcs {
 			var fd descriptorpb.FileDescriptorProto
 			if err := prototext.Unmarshal([]byte(src), &fd); err != nil {
-				t.Fatalf("proto.UnmarshalText(%s, &fd) failed with %v; want success", src, err)
+				t.Fatalf("proto.UnmarshalText(%s, &file) failed with %v; want success", src, err)
```

Comment:
Was this update accidental?

Reply:
Yes :) I will revert it. Initially, I approached this PR a bit differently but later figured out it can be done with fewer changes.